docs/new-feature-tutorial.md

```diff
@@ -50,13 +54,17 @@ organization in Zulip). The following files are involved in the process:
 **Update the model:** The server accesses the underlying database in
 `zerver/models.py`. Add a new field in the appropriate class.
 
-**Create and run the migration:** To create and apply a migration, run:
+**Create and run the migration:** To create and apply a migration, run the
+following commands:
 
 ```
 ./manage.py makemigrations
 ./manage.py migrate
 ```
 
+Zulip uses Django's migration process for creating and
```

Comment:
We should link to https://docs.djangoproject.com/en/1.11/topics/migrations/ here.